recipes/Axelrod/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "Axelrod" %}
+{% set version = "2.10.0" %}
+{% set sha256 = "9c5ecf5e6b5a9f8931888ec8305de7814585d9a10c75ad18cde93a80b8e45fa6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  
+  run:
+    - python
+    - numpy >=1.9.2
+    - matplotlib >=1.4.2
+    - hypothesis >=3.2
+    - tqdm >=3.4.0
+    - prompt_toolkit >=1.0.7
+
+test:
+  imports:
+    - axelrod
+    - axelrod.data
+    - axelrod.strategies
+    - axelrod.tests
+
+about:
+  home: http://github.com/Axelrod-Python/Axelrod
+  license: MIT
+  license_file: LICENSE.txt
+  summary: 'Research tool for Iterated Prisoners Dilemma in Python'
+
+  description: |
+    Axelrod is a library for conducting research in Iterated Prisoner's
+    Dilemma and enable reproducibilty of research on Iterated Prisoner's
+    Dilemma. The library contains over 150 strategies for IPD and supports 
+    Python 3.5 and 3.6
+  doc_url: http://axelrod.readthedocs.io
+  dev_url: https://github.com/Axelrod-Python/Axelrod
+
+extra:
+  recipe-maintainers:
+    - drvinceknight
+    - marcharper
+    - meatballs
+    - souravsingh
```

Comment:
@souravsingh I trust you contacted everybody above and checked if they are OK to be maintainers of this recipe.